src/TestSuite/IntegrationTestCase.php

```diff
@@ -702,39 +702,51 @@ public function viewVariable($name)
      *
      * @return void
      */
-    public function assertResponseOk()
+    public function assertResponseOk($message)
```

Comment:
These new parameters have to be optional parameters for backwards compatibility. `$message = null` would be a good default.